src/config.js

```diff
@@ -7,6 +7,8 @@ const DISCORD_SERVER = {
   discordClientId: process.env.DISCORD_CLIENT_ID,
   discordGuildId: process.env.DISCORD_GUILD_ID,
   botLanguage: process.env.DISCORD_LANGUAGE || 'en',
+  discordAnnouncementsChannel: process.env.DISCORD_ANNOUNCEMENTS_CHANNEL_ID,
+  scheduledDiscordEventsEnabled: process.env.SCHEDULED_DISCORD_EVENTS_ENABLED,
```

Comment:
as it is an string it would always be true. add `=== "true"` to convert to a boolean
```suggestion
  scheduledDiscordEventsEnabled: process.env.SCHEDULED_DISCORD_EVENTS_ENABLED === "true",
```

Reply:
That´s right thanks!